lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -72,7 +72,7 @@ public IMRUJobDefinitionBuilder()
             _updateTaskMemory = 512;
             _coresPerMapper = 1;
             _updateTaskCores = 1;
-            _maxRetryNumberInRecovery = 0;
+            _maxRetryNumberInRecovery = -1;
```

Comment:
For IMRU, I think the default is to support retry. For IMRUExample and our own functional tests, it is easy to set it from client side. For our big data testing, currently, client use default. Of course, we can ask them to change. But just make our test easy without change anything.